### ISSUE TITLE
Compound Slots not working with variants

### ISSUE THREAD

**Issue by rawnly** [nextui-org/tailwind-variants#74]
Given the following configuration

```tsx
const grid = tv({
  variants: {
    documents: {
      true: {},
    }
  },
  defaultVariants: {
    documents: true
  },
  slots: {
    general: 'col-span-7 row-span-4 card',
    chart: 'card !px-0 col-span-5 relative row-span-4',
    documents: 'row-span-6 col-span-5 card',
    flow: 'relative row-span-6 card'
  },
  compoundSlots: [
    // {
    //   slots: ['documents', 'chart', 'general', 'flow'],
    //   className: 'bg-violet-10',
    //   documents: false
    // },
    {
      documents: false,
      slots: ['documents'],
      className: 'hidden'
    }, {
      documents: false,
      slots: ['flow'],
      className: 'col-span-12'
    }, {
      documents: true,
      slots: ['flow'],
      className: 'col-span-7'
    }
  ]
})

// usage later
const { general, flow, documents, chart } = grid({ documents: true })

// i'd expect to have `col-span-12` applied below
// jsx
<div className={flow()} />
```

The variants are never applied 🤷‍♂️  (version `0.1.10`).
Maybe I misunderstood how to use them?

**Comment by Rawnly** [comment]
Also in [the docs][0] the `size` is never specified in the example code

[0: https://www.tailwind-variants.org/docs/slots#compound-slots

**Comment by viktorbonino** [comment]
I also have the same problem.

**Comment by mskelton** [comment]
I have a fix for this in #83
